articles/tutorials/building_2d_games/06_optimizing_texture_rendering/index.md

```diff
@@ -0,0 +1,503 @@
+| `Width`           | `float`               | The width of the sprite, calculated by multiplying the width of the `_sourceRectangle` by the x-axis scale factor.   |
+| `Height`          | `float`               | The height of the sprite, calculated by multiplying the height of the `_sourceRectangle` by the y-axis scale factor. |
+
+| Method                         | Returns  | Description                                                                                              |
+| ------------------------------ | -------- | -------------------------------------------------------------------------------------------------------- |
+| `Sprite(Texture2D, Rectangle)` | `Sprite` | Creates a new instance of the `Sprite` class using source texture and source rectangle parameters given. |
+| `Draw(SpriteBatch, Vector2)`   | `void`   | Draws the sprite using the [**SpriteBatch**](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch) provided at the specified position.                       |
+
+> [!NOTE]
+> The properties of the `Sprite` class directly correspond to the parameters used in [**SpriteBatch.Draw**](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch.Draw(Microsoft.Xna.Framework.Graphics.Texture2D,Microsoft.Xna.Framework.Vector2,System.Nullable{Microsoft.Xna.Framework.Rectangle},Microsoft.Xna.Framework.Color,System.Single,Microsoft.Xna.Framework.Vector2,Microsoft.Xna.Framework.Vector2,Microsoft.Xna.Framework.Graphics.SpriteEffects,System.Single)). This design makes it simple to encapsulate all rendering information while maintaining flexibility in how each sprite is displayed.
+
+> [!TIP]
+> The `Width` and `Height` properties automatically account for scaling, making it easier to perform calculations like collision detection or positioning without manually applying the scale factor each time.
+
+## Adding the `Sprite` class
+
+Now that we have defined what the `Sprite` class should be, let's create it.  Instead of adding this to the game project, we're going to add it to the *MonoGameLibrary* class library project that was setup in [Chapter 04](../04_game_library/index.md).  Perform the following:
+
+1. Add a new folder to the *MonoGameLibrary* project named *Graphics*.
+2. Create a new file named *Sprite.cs* in that folder.
+
+Add the following code for the foundation of the `Sprite` class to the *Sprite.cs* file:
+
+```cs
+using System.Diagnostics;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace MonoGameLibrary.Graphics;
+
+public class Sprite
+{
+
+}
+```
+
+> [!NOTE]
+> The *Sprite.cs* class file is placed in the *MonoGame/Graphics* directory and the class uses the `MonoGame.Graphics` namespace to keep rendering-related classes organized together.  As we add more functionality to the library, we'll continue to use directories and namespaces to maintain a clean structure.
```

Comment:
Include a link to the MSDN docs on what a namespace is (on the word namespace :D )

Reply:
Link to MSDN article on namespaces added